rudder-web/src/main/scala/bootstrap/liftweb/AppConfigAuth.scala

```diff
@@ -166,6 +170,58 @@ class AppConfigAuth extends Loggable {
 
 }
 
+/*
+ * A trivial extension to the SimpleUrlAuthenticationFailureHandler that correctly log failure
+ */
+class RudderUrlAuthenticationFailureHandler(failureUrl: String) extends SimpleUrlAuthenticationFailureHandler(failureUrl) {
+  override def onAuthenticationFailure(request: HttpServletRequest, response: HttpServletResponse, exception: AuthenticationException): Unit = {
+    LogFailedLogin.warn(exception, request)
+    super.onAuthenticationFailure(request, response, exception)
+  }
+}
+
+object LogFailedLogin {
+
+  def warn(ex: AuthenticationException, request: HttpServletRequest): Unit = {
+    ApplicationLogger.warn(s"Login authentication failed for user '${getUser(ex)}' from IP '${getRemoteAddr(request)}': ${ex.getMessage}")
```

Comment:
I looked, and we don't get very much interesting information with that:
```
[2017-02-24 17:53:33] WARN  application - Login authentication failed for user 'admin' from IP '0:0:0:0:0:0:0:1': Bad credentials on url: http://localhost:8082/rudder-web/j_spring_security_check
```
I'm really not sure we want to add it.